test-integration/helpers/ui/test-element.coffee

```diff
@@ -28,14 +32,28 @@ class TestItemHelper
     else
       @locator
 
-  get: (args...) =>
+  aliasWait: (waitFunction, args...) =>
     locator = @getLocator(args...)
-    @test.utils.wait.for(locator)
+
+    waitTime = _.last(args)
```

Comment:
I think we should be able to instead of re-writing existing functions with one variable changed.  since these are aliases, it seems like args should be able to pass through.  Also, by collecting these things into one `wait.for` we ensure that all the logging from that doesn't get neglected.